src/features/content-answers/ContentAnswersOpenButton.tsx

```diff
@@ -12,12 +13,22 @@ import './ContentAnswersOpenButton.scss';
 interface ContentAnswersOpenButtonProps {
     intl: IntlShape;
     onClick: () => void;
+    fileExtension: string;
 }
 
-const ContentAnswersOpenButton = ({ intl, onClick }: ContentAnswersOpenButtonProps) => {
+const ContentAnswersOpenButton = ({ intl, onClick, fileExtension }: ContentAnswersOpenButtonProps) => {
     const { formatMessage } = intl;
 
+    const isAllowedFileType = (extension: string) => {
+        const allowedTypes = CODE_FILE_EXTENSIONS.concat(DOCUMENT_FILE_EXTENSIONS).concat(TEXT_FILE_EXTENSIONS);
```

Comment:
curious your thoughts on this:
```
        const allowedTypes = [
            ...CODE_FILE_EXTENSIONS,
            ...DOCUMENT_FILE_EXTENSIONS,
            ...TEXT_FILE_EXTENSIONS,
        ];
```

I've seen spread operator more common than `concat` in modern js but not sure if there's an objective reasons for using one over the other. thoughts?

Reply:
im fine with changing it, this was copied over